src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 40 * days;
+  }
+
+  if (days < 7) {
+    return (40 * days) - 20;
+  }
+
+  return (40 * days) - 50;
```

Comment:
all these numbers should be in variables, keep your code without "magic" numbers, I don't know what numbers `40, 20, 50` mean, you must explain it to me in the code

Reply:
got it, sir